<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieList</class>
 <widget class="QMainWindow" name="MovieList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movie List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Movie List</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelMovieID">
        <property name="text">
         <string>Movie ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="movieIDLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelDate">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelCategory">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QListWidget" name="categoryListWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <item>
         <property name="text">
          <string>Adventure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Children</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fantasy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comedy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drama</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sci-Fi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Romance</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUpdate">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelect">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveToExcel">
        <property name="text">
         <string>Save to Excel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoadFromExcel">
        <property name="text">
         <string>Load from Excel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="movieTableWidget">
      <column>
       <property name="text">
        <string>Movie ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
